### PR TITLE
Update office-365-worldwide-endpoints.md

### LINKED ISSUE #12099: Include note for Cross Cloud Anonymous Join for the Teams endpoint section
Teams in WW supports Cross-Cloud Anonymous Join, which means a WW Teams user/client can join DoD or GCCH hosted meetings. This has caused some confusion/lead to customers raising cases because they have whitelisted for WW, but not for the other clouds. It is currently only implied that if they want to be able to connect to GCCH/DoD meetings, that they have to whitelist for those endpoints.

Could we please include a warning/notification under the Teams subsection that for Cross-Cloud Join to properly work, they must whitelist the endpoints for the cloud of the target meeting? Also add links to the other cloud endpoint pages.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 66bd8666-3566-a901-b25b-eb6baba6264a
* Version Independent ID: e9f6310f-2207-8c81-d31a-3d5a80beb26b
* Content: [Office 365 URLs and IP address ranges - Microsoft 365 Enterprise](https://learn.microsoft.com/en-us/microsoft-365/enterprise/urls-and-ip-address-ranges?view=o365-worldwide)
* Content Source: [microsoft-365/enterprise/urls-and-ip-address-ranges.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/enterprise/urls-and-ip-address-ranges.md)
* Service: **microsoft-365-enterprise**
* GitHub Login: @kelleyvice-msft
* Microsoft Alias: **kvice**

### PR DESCRIPTION
**Pull request by msbemba**
added a note for teams cross-cloud join

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12099

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/includes/office-365-worldwide-endpoints.md [hunk 1]
```diff
@@ -45,8 +45,15 @@ ID | Category | ER | Addresses | Ports
...
 180 | Default<BR>Required | No | `compass-ssl.microsoft.com` | **TCP:** 443
-
-## Microsoft 365 Common and Office Online
+  
+Notes for this table:
+
+For  Cross-Cloud Join to properly work, you must whitelist the endpoints for the cloud of the target meeting
+  
+  [Office 365 U.S. Government GCC High endpoints](/microsoft-365/enterprise/microsoft-365-u-s-government-gcc-high-endpoints)
+  [Office 365 U.S. Government DoD endpoints](/microsoft-365/enterprise/microsoft-365-u-s-government-dod-endpoints)
```
Comment (JohanFreelancer9) [inline microsoft-365/includes/office-365-worldwide-endpoints.md]:
```suggestion
> [!NOTE]
> For  Cross-Cloud Anonymous Join to work properly, you must add these endpoints for the cloud of the target meeting to the safe senders list:
> - [Office 365 URLs and IP address ranges](/microsoft-365/enterprise/urls-and-ip-address-ranges)
> - [Office 365 U.S. Government DoD endpoints](/microsoft-365/enterprise/microsoft-365-u-s-government-dod-endpoints)

```

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kelleyvice-msft Please merge this PR. Thanks

**Comment by bradbrill** [comment]
@yogkumgit I don't think kelleyvice is available. Is there an alternative contact we can get approval from?

**Pull request merged**